controllers/drplacementcontrol.go

```diff
@@ -84,7 +84,7 @@ func (d *DRPCInstance) startProcessing() bool {
 }
 
 func (d *DRPCInstance) processPlacement() (bool, error) {
-	d.log.Info("Process DRPC Placement", "DRAction", d.instance.Spec.Action)
+	d.log.V(1).Info("Process DRPC Placement", "DRAction", d.instance.Spec.Action)
```

Comment:
Not clear why do we need `.V(1)` when we already use `.Info()`. Maybe this is a limit of the logger we use, but normally when you use `.Info()` the logger logs a message in INFO level and the log is available if the log level is equal or higher than INFO level.

Maybe this code means that this is a DEBUG level log - in this case we should have something like `.Debug()` call to log the issue.